core/core.go

```diff
@@ -427,6 +427,21 @@ func (c *Core) RenameSCFolder(tempFolderPath string, smartContractName string) (
 	}
 	return scFolderName, err
 }
+func (c *Core) Foldercheck(smartContractName string) interface{} {
```

Comment:
Rename the function to `IsSmartContractDirectoryPresent` and change the return type to `bool`